File: QUERA QUESTIONS SOLVED/Kamel Boodan Ya Naboodn.cpp
#include<iostream>
#include<cmath>
using namespace std;
int main(){
    int num;
    cin >> num;
    int i=1 , sum=0 , r;
    while(i < num){
        r = num%i;
        if(r==0){
            sum += i;
        }
        i++;
    }
    if(sum==num){
        cout << "YES";
    }
    else{
        cout << "NO";
    }
    return 0;  
} 

File: QUERA QUESTIONS SOLVED/Kamel Boodan Ya Naboodn.py
import math
num = int(input())
copyNum = num 
sum = 0
while copyNum > 1:
    for i in range(2,int(math.sqrt(num))+2,1):
        while copyNum % i==0:
            copyNum= copyNum // i
            sum+=copyNum
    
if sum==num:
    print("YES")

else:
    print("NO")